lib/logstash/outputs/syslog.rb

```diff
@@ -94,8 +94,12 @@ class LogStash::Outputs::Syslog < LogStash::Outputs::Base
   # syslog message format: you can choose between rfc3164 or rfc5424
   config :rfc, :validate => ["rfc3164", "rfc5424"], :default => "rfc3164"
 
+  # use codec to format message (instead of setting message)
+  config :usecodec, :validate => :boolean, :default => false
```

Comment:
I agree, this should just use codec, if it's not defined, then default to something that produces the traditional output (ideally the codec plain with the format of message)